src/pages/evm/staking/StakingPageHeader.vue

```diff
@@ -92,7 +163,7 @@ const secondLineData = computed(() => [{
 // methods
 onBeforeMount(() => {
     chainSettings.getUsdPrice().then((price) => {
```

Comment:
Does it make sense to do some error handling here?

Reply:
The whole onBeforeMount code block is unused and can be deleted.
Thanks!